common/v2/components/TransactionFlow/TransactionReceipt.tsx

```diff
@@ -90,21 +96,24 @@ export default function TransactionReceipt({
     txConfig.receiverAddress,
     txConfig.network
   );
-  const recipientLabel = recipientContact ? recipientContact.label : 'Unknown Address';
+  const recipientLabel = recipientContact ? recipientContact.label : translateRaw('NO_ADDRESS');
 
   /* ToDo: Figure out how to extract this */
   const { asset, gasPrice, gasLimit, senderAccount, network, data, nonce, baseAsset } = txConfig;
 
   /* Determing User's Contact */
   const senderContact = getContactByAccount(senderAccount);
-  const senderAccountLabel = senderContact ? senderContact.label : 'Unknown Account';
+  const senderAccountLabel = senderContact ? senderContact.label : translateRaw('NO_LABEL');
 
   const localTimestamp = new Date(Math.floor(timestamp * 1000)).toLocaleString();
   const assetAmount = displayTxReceipt.amount || txConfig.amount;
   const assetTicker = 'asset' in displayTxReceipt ? displayTxReceipt.asset.ticker : 'ETH';
   const assetForRateFetch = 'asset' in displayTxReceipt ? displayTxReceipt.asset : undefined;
 
   const txUrl = displayTxReceipt.network.blockExplorer.txUrl(displayTxReceipt.hash);
+  const shouldRenderPendingBtn =
+    pendingButton && txStatus === ITxStatus.PENDING && !isWeb3Wallet(senderAccount.wallet);
```

Comment:
We'll want to also disallow WALLETCONNECT wallet type

Reply:
WalletConnect's wallet type is also WEB3 (from PR: https://github.com/MyCryptoHQ/MyCrypto/pull/3024/files#diff-29db5304b05c462e07ad1bd0dab2df7dR211) and it should be covered automatically after PR is merged due to using `isWeb3Wallet` function which checks if `wallet.type === WEB3`.